interfaces/repo_test.go

```diff
@@ -578,9 +578,16 @@ func (s *RepositorySuite) TestConnectSucceedsWhenIdenticalConnectExists(c *C) {
 	err = s.testRepo.Connect(s.plug.Snap, s.plug.Name, s.slot.Snap, s.slot.Name)
 	c.Assert(err, IsNil)
 	// Only one "connect" is actually present.
-	c.Assert(s.testRepo.ConnectedSlots(s.slot.Snap), DeepEquals, map[*Slot][]*Plug{
-		s.slot: []*Plug{s.plug},
-	})
+	c.Assert(s.testRepo.Interfaces().Slots[0].Snap, Equals, s.slot.Snap)
+	c.Assert(s.testRepo.Interfaces().Slots[0].Name, Equals, s.slot.Name)
+	c.Assert(s.testRepo.Interfaces().Slots[0].Connections, HasLen, 1)
+	c.Assert(s.testRepo.Interfaces().Slots[0].Connections[0].Snap, Equals, s.plug.Snap)
+	c.Assert(s.testRepo.Interfaces().Slots[0].Connections[0].Name, Equals, s.plug.Name)
+	c.Assert(s.testRepo.Interfaces().Plugs[0].Snap, Equals, s.plug.Snap)
+	c.Assert(s.testRepo.Interfaces().Plugs[0].Name, Equals, s.plug.Name)
+	c.Assert(s.testRepo.Interfaces().Plugs[0].Connections, HasLen, 1)
+	c.Assert(s.testRepo.Interfaces().Plugs[0].Connections[0].Snap, Equals, s.slot.Snap)
+	c.Assert(s.testRepo.Interfaces().Plugs[0].Connections[0].Name, Equals, s.slot.Name)
```

Comment:
Go has that concept of "variables" that allow you to do something magic like this:

    slot := s.testRepo.Interfaces().Slots[0]

and then use it over and over! :-)